src/main/java/org/humancellatlas/ingest/core/MetadataDocumentMessageBuilder.java

```diff
@@ -124,4 +135,10 @@ public ExportMessage buildAssaySubmittedMessage() {
                 documentType.getSimpleName(), envelopeId, envelopeUuid, assayIndex, totalAssays);
     }
 
+    public ExportMessage buildUpdateExportMessage(BundleManifest bundleManifest) {
+        String callbackLink = linkGenerator.createCallback(bundleManifest.getClass(),bundleManifest.getId());
+        return new ExportMessage(UUID.fromString(bundleManifest.getBundleUuid()), DateTime.now().toString(), messageProtocol, metadataDocId, metadataDocUuid, callbackLink,
```

Comment:
@aaclan-ebi we need information about the UUIDs to be updated here. Can you extend the ExportMessage, or maybe make a similar ExportUpdateMessage that contains an array of callbackLinks?

Reply:
Done. Sample bundle update message:
```{
	"bundleUuid": "72de15ec-93fa-4421-b01e-59a4f9b32b46",
	"versionTimestamp": "2019-05-17T17:28:42.243+01:00",
	"callbackLinks": ["/processes/5c2dfb171603f500078b2922", "/protocols/5c2dfb171603f500078b2915", "/files/5c2dfb161603f500078b28e9", "/processes/5c2dfb171603f500078b2917", "/biomaterials/5c2dfb161603f500078b28fc", "/protocols/5c2dfb171603f500078b2914", "/processes/5c2dfb171603f500078b2924", "/biomaterials/5c2dfb171603f500078b28ff", "/biomaterials/5c2dfb161603f500078b28f8", "/processes/5c2dfb171603f500078b291f", "/processes/5c2dfb171603f500078b2921", "/biomaterials/5c2dfb161603f500078b28f7", "/processes/5c2dfb171603f500078b2923", "/biomaterials/5c2dfb161603f500078b28fb", "/processes/5c2dfb171603f500078b2930", "/biomaterials/5c2dfb171603f500078b2902", "/biomaterials/5c2dfb171603f500078b28fd", "/files/5c2dfb161603f500078b28e1", "/files/5c2dfb161603f500078b28e3", "/biomaterials/5c2dfb161603f500078b28fa", "/biomaterials/5c2dfb171603f500078b2905", "/processes/5c2dfb171603f500078b2920", "/biomaterials/5c2dfb171603f500078b290b", "/protocols/5c2dfb171603f500078b2912", "/files/5c2dfb161603f500078b28e7", "/biomaterials/5c2dfb171603f500078b2900", "/files/5c2dfb161603f500078b28e8", "/biomaterials/5c2dfb171603f500078b2901", "/biomaterials/5c2dfb171603f500078b2908", "/files/5c2dfb161603f500078b28e2", "/processes/5c2dfb171603f500078b292d", "/projects/5c2dfb161603f500078b28e0", "/biomaterials/5c2dfb171603f500078b290e", "/biomaterials/5c2dfb161603f500078b28f9", "/processes/5c2dfb171603f500078b291d", "/protocols/5c2dfb171603f500078b2913", "/processes/5c2dfb171603f500078b291c", "/biomaterials/5c2dfb171603f500078b28fe", "/protocols/5c2dfb171603f500078b2911", "/processes/5c2dfb171603f500078b291e", "/processes/5c2dfb171603f500078b2927", "/biomaterials/5c2dfb161603f500078b28f6", "/processes/5c2dfb171603f500078b292a"],
	"envelopeId": "5c2dfb101603f500078b28de",
	"envelopeUuid": "4e474a77-1489-42ef-b0a7-6290c2cfce29",
	"index": 0,
	"total": 12,
	"messageProtocol": null
}```